.gitignore

```diff
@@ -23,8 +23,6 @@ pnpm-debug.log*
 *.sln
 *.sw?
 _output
-# sub chart tgz
-charts/*/charts
```

Comment:
This also needs to be committed to the repository, making it convenient for users to install directly.

Reply:
I remember that Karmada had submitted this to the repo before, but it seems I was mistaken.